playbooks/lets-encrypt.yml

```diff
@@ -0,0 +1,58 @@
+---
+# lets-encrypt.yml asks questions about Streisand domain name, which is used to
+# request TLS certificates from Let's Encrypt server. If user fails to answer
+# these questions, certain roles are disabled.
+- name: Collect information about the Streisand domain
+# =========================================
+  hosts: streisand-host
+  gather_facts: yes
```

Comment:
Do you need `gather_facts` here? I suspect it might not be necessary

Reply:
@cpu Ah I missed this comment. This is why the CI build fails.